content/en/docs/setup/upgrade/canary/canary_upgrade_test.sh

```diff
@@ -55,4 +55,4 @@ _verify_like snip_uninstall_old_control_plane_3 "$snip_uninstall_old_control_pla
 
 # @cleanup
 snip_uninstall_canary_control_plane_1
-snip_cleanup_1
\ No newline at end of file
+snip_cleanup_2
```

Comment:
why removed 1? I think we should have all 3 snip cleanups?

Reply:
This is not removed, cleanup_2 is the old cleanup_1, as I've added a new cleanup_1 which is doing `tag remove`. In this test, there's nothing related to the tag, so I didn't add the tag cleanup.

Reply:
I see 1 and 3 being called in revision_tags_test so all 3 are being called at some point.